From eda9826d25336bcf661700270c580d4d62128750 Mon Sep 17 00:00:00 2001
From: Richard W.M. Jones <rjones@redhat.com>
Date: Fri, 4 Mar 2011 12:13:32 +0000
Subject: [PATCH 2/2] virt-make-fs: Round disk size to integer, fix for qemu-img 0.14.

qemu-img used to allow you to specify a fractional image size in bytes
(or at least, it used to ignore the part after the decimal place).  In
qemu-img 0.14 it no longer does this so we round down the size to a
whole number of bytes.
---
 tools/virt-make-fs |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/tools/virt-make-fs b/tools/virt-make-fs
index 3825f70..833c8a5 100755
--- a/tools/virt-make-fs
+++ b/tools/virt-make-fs
@@ -393,6 +393,8 @@ if (!defined $size) {
     }
 }
 
+$size = int ($size);
+
 # Create the output disk.
 # Take the unusual step of invoking qemu-img here.
 
-- 
1.7.4

